packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -117,6 +117,9 @@ export async function processHttpRequest(
     level: parsedConfig.nodeSettings.logLevel,
   });
 
+  // Unlike GCP, AWS automatically generates a unique request ID so no need to use one here
+  logger.info(`Received HTTP gateway request`);
+
```

Comment:
Is this actually necessary? I think every request (lambda invocation) is logged in the AWS (and I think also in GCP) anyway. I don't think this adds anything. I would at least set it to `debug`.

Reply:
👍 I've removed it for AWS given AWS automatically assigns a unique RequestID. For GCP, given the different layout of logs, I've changed it to `debug`.